usr/share/rear/restore/GALAXY11/default/400_restore_with_galaxy.sh

```diff
@@ -0,0 +1,86 @@
+#
+# restore data with Commvault
+#
+
+# verify that we have a backupset
+test "$GALAXY11_BACKUPSET" || Error "Commvault Backup Set not defined [GALAXY11_BACKUPSET=]."
+
+# create argument file
+
+cat <<EOF >$TMP_DIR/commvault.restore.options
+$(test -r "$GALAXY11_Q_ARGUMENTFILE" && cat "$GALAXY11_Q_ARGUMENTFILE")
+[sourceclient]
+$HOSTNAME
+[level]
+1
+[options]
+QR_PRESERVE_LEVEL
+QR_DO_NOT_OVERWRITE_FILE_ON_DISK
+$GALAXY11_PIT
+[dataagent]
+Q_LINUX_FS
+[backupset]
+$GALAXY11_BACKUPSET
+[sourcepaths]
+/
+[destinationpath]
+$TARGET_FS_ROOT
+EOF
+
+if [ "x$GALAXY11_ZEIT" != "x" ]; then
+	cat <<EOF >>$TMP_DIR/commvault.restore.options
+[browseto]
+$GALAXY11_ZEIT
+EOF
+fi
+
+local jobstatus=Unknown
+
+if jobid=$(qoperation restore -af $TMP_DIR/commvault.restore.options); then
+	jobid=${jobid// /} # remove trailing blanks
+	prevstatus=
+
+	LogPrint "Restoring data with Commvault (job $jobid)"
+
+	while true; do
+		sleep 60
+		# output of qlist job -co sc -j ## :
+
+		# STATUS     COMPLETE PERCENTAGE
+		# ------     -------------------
+		# Running
+		jobstatus=$(qlist job -j $jobid -co sc | tail -n 1)
+
+		# stop waiting if the job reached a final status
+		case "$jobstatus" in
+		?omplet*)
```

Comment:
This is nitpicking coding style only (feel free to ignore)
I guess it comes from copied old code here:

Usually we do:
* Indentation with 4 blanks, no tabs.
* Use paired parentheses for case patterns so that editor commands (like '%' in 'vi') that check for matching opening and closing parentheses work everywhere in the code.

cf. https://github.com/rear/rear/wiki/Coding-Style